components/Chat/ChatSettings/ChatSettingsPanel/ChatGeneralSettingsPanel.vue

```diff
@@ -5,11 +5,12 @@
     class="mb-3 px-3"
   >
     <!-- General Settings Toggles -->
-    <div>
+    <div id="settings">
```

Comment:
Maybe `#chat-settings` instead of `#settings` since there may be other settings / other settings menus in the future.